tensorboard/webapp/app_routing/effects/app_routing_effects_test.ts

```diff
@@ -234,6 +238,18 @@ describe('app_routing_effects', () => {
     spyOn(store, 'dispatch').and.callFake((action: Action) => {
       actualActions.push(action);
     });
+
+    const mockRandomValuesFunction = function <Uint8Array>(
+      arr: Uint8Array
+    ): Uint8Array {
+      if (arr instanceof Uint8Array) {
+        arr.set(TEST_UINT8ARRAY);
+      }
+      return arr;
```

Comment:
You could consider throwing an Error here instead of silently doing nothing. It would make clearer that something unexpected has happened.